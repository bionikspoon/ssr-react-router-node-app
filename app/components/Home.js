import React from 'react'

export default function Home() {
  return <h1>You are on your home page</h1>
}
