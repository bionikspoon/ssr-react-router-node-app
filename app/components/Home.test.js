import React from 'react'
import { mount } from 'enzyme'

import Home from './Home'

describe('components/Home', () => {
  test('it should render correctly/', () => {
    const component = mount(<Home />)

    expect(component).toMatchSnapshot()
  })
})
