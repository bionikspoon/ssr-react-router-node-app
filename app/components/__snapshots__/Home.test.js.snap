// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Home it should render correctly/ 1`] = `
ReactWrapper {
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": Object {
      "instance": <h1>
        You are on your home page
      </h1>,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "You are on your home page",
      },
      "ref": null,
      "rendered": Array [
        "You are on your home page",
      ],
      "type": "h1",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": Object {
        "instance": <h1>
          You are on your home page
        </h1>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "You are on your home page",
        },
        "ref": null,
        "rendered": Array [
          "You are on your home page",
        ],
        "type": "h1",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
